src/components/Selector/CertificationSelector.tsx

```diff
@@ -0,0 +1,329 @@
+import { SmallLoadingSpinner } from '@app/components/Common/LoadingSpinner';
+import defineMessages from '@app/utils/defineMessages';
+import type { Region } from '@server/lib/settings';
+import React, { useCallback, useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
+import AsyncSelect from 'react-select/async';
+import useSWR from 'swr';
+
+interface Certification {
+  certification: string;
+  meaning?: string;
+  order?: number;
+}
+
+interface CertificationResponse {
+  certifications: {
+    [country: string]: Certification[];
+  };
+}
+
+interface CertificationOption {
+  value: string;
+  label: string;
+  certification?: string;
+}
+
+interface CertificationSelectorProps {
+  type: string;
+  certificationCountry?: string;
+  certification?: string;
+  certificationGte?: string;
+  certificationLte?: string;
+  onChange: (params: {
+    certificationCountry?: string;
+    certification?: string;
+    certificationGte?: string;
+    certificationLte?: string;
+  }) => void;
+  showRange?: boolean;
+}
+
+const messages = defineMessages('components.CertificationSelector', {
```

Comment:
This should be `components.Selector.CertificationSelector`
```suggestion
const messages = defineMessages('components.Selector.CertificationSelector', {
```